summary_workflows/quantification/APAeval_Q_test_data/test.pd.mm10_test.bed

```diff
@@ -1,10 +1,12 @@
-chr1	2199734	2199735	not_in_pd	156.240540124	-
```

Comment:
renaming the file is fine, but you need to adjust the config such that the test runs as expected. `https://github.com/iRNA-COSI/APAeval/blob/6a00c0428c4c8e970e17e06c3739328a0ea0a935/summary_workflows/quantification/nextflow.config#L40`